Documentation improvement (#101)
lxylxy123456/ngshare: docs/contributer_guide/vngshare.rst

vngshare
========

> vngshare is the stand-alone mode version of ngshare. It stands for Vserver-like Notebook Grader Share. It is similar to `vserver <https://github.com/lxylxy123456/ngshare-vserver/>`_ and allows easy testing.

Reviewer: ```suggestion
vngshare is the stand-alone mode of ngshare. It stands for Vserver-like Notebook Grader Share. It is similar to `vserver <https://github.com/lxylxy123456/ngshare-vserver/>`_ and allows easy testing.
```

Install
-------
For detailed instructions, see :doc:`install`.

.. code:: bash

    pip3 install tornado jupyterhub sqlalchemy
    cd ngshare
    python3 vngshare.py [--host <bind_IP_address> [--port <port_number>]]

Default Behavior
----------------

> vngshare by default enables debug (e.g. verbose error output). It allows developers to view and reset database content easily. Users can be authenticated by just passing the username in GET / POST requests (see :doc:`/api/authentication`).

Reviewer 2: ```suggestion
vngshare by default enables debug (e.g. verbose error output). It allows developers to view and reset database content easily. Users can be authenticated by simply passing in their username in GET / POST requests (see :doc:`/api/authentication`).
```
